File: PythonSelenium/PythonApplication1/src/pageobjects/LoginPage.py
from selenium import webdriver
from selenium.webdriver.common.by import By

class LoginPage():
    def enter_email_address(driver, emailAddress):
        driver.find_element(By.ID, "validateEmail").click()
        driver.find_element(By.ID, "validateEmail").clear()
        driver.find_element(By.ID, "validateEmail").send_keys(emailAddress)
        driver.find_element(By.ID, "validateForm").submit()
        #driver.find_element(By.ID, "validateAccountButton").click()

    def enter_password(driver, password):
        driver.find_element(By.ID, "password").clear()
        driver.find_element(By.ID, "password").send_keys()
        driver.find_element(By.CSS_SELECTOR, "#loginAndCheckoutButton > div.text").click()

File: Seleniulcsharp/Seleniulcsharp/src/pageobjects/LoginPage.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Seleniulcsharp.src.pageobjects
{
    internal class LoginPage
    {
        public void enter_email_address(IWebDriver driver, string emailAddress)
        {
            driver.FindElement(By.Id("validateEmail")).Click();
            driver.FindElement(By.Id("validateEmail")).Clear();
            driver.FindElement(By.Id("validateEmail")).SendKeys(emailAddress);
            driver.FindElement(By.Id("validateForm")).Submit();
        }

        public void enter_password(IWebDriver driver, string password)
        {
            driver.FindElement(By.Id("password")).Clear();
            driver.FindElement(By.Id("password")).SendKeys(password);
            driver.FindElement(By.CssSelector("#loginAndCheckoutButton > div.text")).Click();
        }
    }
}
